### ISSUE TITLE
Using Genesys Dice in the fantasy statblock

### ISSUE THREAD

**Issue by GradientForce** [Craftidore/Obsidian-Genesys#12]
Hello, I was wondering if there was a way to have the genesys dice populate in the statblock under abilities. I know that the skills will populate dice, but if I wanted to add an ability like field commander, and the associated text, I'd like to make the dice for the difficulty of the check show up.
![image](https://github.com/user-attachments/assets/89459428-85e0-43ec-bf8f-5e5f6e54feb8)
I tried using #Genesysdifficulty, but that didn't work.

**Comment by source-toad** [comment]
Yeah there sure is, it just depends on the dice snippet you’ve got installed. If you’re using the correct tag, make sure you’ve wrapped the description in quotes (“”).
